package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-input-color",
-  "version": "4.0.0",
+  "version": "4.1.0",
```

Comment:
Don't bump version in this pull request, I will increase it when releasing from the main branch

Reply:
Should be good to go now...